### DISCUSSION TITLE
Regarding IBM for Wind Turbine Case

### DISCUSSION THREAD

**Discussion by dineshkumarreddyg36** [xcompact3d/Incompact3d#107] [Q&A] [answered]
I wish to run the **Wind Turbine** case **(NREL-5MW_ALM)** using **Immersed Boundary Method** instead of **ALM**. So, I have changed the **flag** for **ibm i.e., iibm=0 to iibm=3** in the **'input.i3d'** file. But this is not working. 
Can Someone please tell what mistake I'm doing?

**Comment by slaizet** [comment]
Hi,

We haven't try to run a wind turbine case purely with IBM as the mesh requirement would be too high and the time step would be too small. If you want to try, you will have to generate the geometry of the turbine and export it to the code, so that the flow reconstructions can be generated. It is not as straightforward as just switching one parameter in the input file.

All the best,

Sylvain

**Reply by dineshkumarreddyg36** [reply]
@slaizet, thanks for the reply.
Could you please tell how to **export a mesh** file to the code?
For example if I've a **mesh file** of a geometry that has been generated in some other **software like ANSYS**, how can I export it to the code?

**Comment by slaizet** [comment, marked as answer]
Hi,

I think I said it already in another discussion but Xcompact3d is based on a Cartesian mesh with uniform rectangular cells. Exporting (unstructured) meshes from another software is not an option. What you need to do is generate the epsilon files to be used for the reconstructions, see for instance [https://github.com/xcompact3d/Incompact3d/blob/master/src/BC-Cylinder.f90].

All the best,

Sylvain

**Reply by dineshkumarreddyg36** [reply]
okay, thanks for the reply @slaizet
